src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -44,6 +44,7 @@
 from ..models import AutoencoderKL
 from ..models.attention_processor import FusedAttnProcessor2_0
 from ..models.modeling_utils import _LOW_CPU_MEM_USAGE_DEFAULT, ModelMixin
+from ..quantizers.bitsandbytes.utils import _check_bnb_status
```

Comment:
Reason why I preferred not having `_check_bnb_status()` inline is because I imagine it'd be used across the library. So, didn't make sense to include it inside of a function.